lib/services/publish.js

```diff
@@ -125,6 +141,11 @@ function publishPageAtUrl(url, uri, data, locals, site) { // eslint-disable-line
   // implementation, but customUrl should override those rules
   url = data.customUrl || url;
 
+  if (data._dynamic) {
+    locals.publishUrlDynamic = true;
```

Comment:
publishUrl is the actual url, right? this seems confusing. maybe `isDynamicPublishUrl` if it's gonna be a bookean?